### PR TITLE
remove password field from Extending Administrate Section

### PR DESCRIPTION
**Pull request by ewertoncodes**
Remove password field from Extending Administrate section. Because password field already exist in the gem administrate.

### CODE REVIEW

**Comment by getaaron** [comment]
I think the `belongs_to_search` could also be removed?

File: docs/extending_administrate.md [hunk 1]
```diff
@@ -8,10 +8,9 @@ number of plugins available, many of which can be found at [RubyGems.org].
 At the time of writing, these appear to be the most popular ones:
```
Comment (pablobm) [inline docs/extending_administrate.md:8]:
```suggestion
These are some popular examples:
```

**Review by pablobm — COMMENTED** [review]
Hello @ewertoncodes, and thank you for the PR. Apologies for the delay.

I think the change makes sense, although it would need to also change the phrasing in the sentence right before the list, as these are not the "most popular ones" any more, but rather a hand picked selection. (See my suggestion).

Perhaps also add a new fifth one, like the jsonb plugin?

Regarding @getaaron's suggestion, I'm not sure it's the same, as the `belongs_to_search` plugin is different from the built-in `belongs_to` field. However the plugin is quite tricky to use (or was last time I checked) so perhaps it's worth removing it too.

Thoughts?

**Comment by ewertoncodes** [comment]
@pablobm and @nickcharlton. Looks like the circle ci needs an update on line:

https://github.com/thoughtbot/administrate/blob/main/.circleci/config.yml#L10.
Can I solve that in this PR?

**Comment by pablobm** [comment]
@ewertoncodes - I just fixed the CI: it required upgrading Rails. Could you please rebase now and I'll merge?

**Comment by ewertoncodes** [comment]
Rebased

**Comment by pablobm** [comment]
Merged! Thank you @ewertoncodes 🙂

**Pull request merged**